fvtest/tril/tril/CMakeLists.txt

```diff
@@ -23,16 +23,19 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake/modules"
 	${CMAKE_MODULE_PATH})
 
 include(OmrUtility)
-include(FindYACC)
-include(FindLEX)
-
-find_package(YACC REQUIRED)
-find_package(LEX REQUIRED)
 
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+find_package(YACC)
+find_package(LEX)
+
+if (YACC AND LEX)
+
+include(FindYACC)
+include(FindLEX)
```

Comment:
I don't think we should include FindYACC and FindLEX. Isn't it equivalent to the find_package({YACC,LEX}) commands right above it?

Also, could you please indent the body of the if/else/endif statement.